<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beállítások</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: #8f8f91;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background-color: rgb(100, 100, 200);
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: white;
	font-size: 16px;
	font-weight: bold;
}</string>
    </property>
    <widget class="QFrame" name="profilePicture">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>191</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 0px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="restoreDefaultResultsButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: white;
	color: grey;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Eredmények törlése</string>
     </property>
    </widget>
    <widget class="QComboBox" name="changeThemeBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="saveAndCloseButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: white;
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: grey;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Mentés</string>
     </property>
    </widget>
    <widget class="QPushButton" name="abortButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>390</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: white;
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: grey;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Mégse</string>
     </property>
    </widget>
    <widget class="QLabel" name="themeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	border: none;
}</string>
     </property>
     <property name="text">
      <string>Téma:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
